recipes/opusfile/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "opusfile" %}
+{% set version = "0.11" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://ftp.osuosl.org/pub/xiph/releases/opus/{{ name }}-{{ version }}.tar.gz
+  sha256: 74ce9b6cf4da103133e7b5c95df810ceb7195471e1162ed57af415fabf5603bf
+  patches:
+    - 0001-Disable-GNU_SOURCE.patch
```

Comment:
Could you please add a comment explaining why this patch is needed?

Reply:
I just saw the `fmilib` package has the same issue, so I added a reference the Github issue as they did.